index.js

```diff
@@ -0,0 +1,15 @@
+let a;
+let b;
+let x;
+
+const divide = function(a, b){
+    return (a / b);
+}
```

Comment:
This can also be one line :D

```suggestion
const divide = (a, b) => a / b;
```